scripts/report-bundle-size.js

```diff
@@ -1,21 +1,33 @@
 #!/usr/bin/env node
 /* eslint-env node */
-
+const fs = require('fs')
 const path = require('path')
 const childProc = require('child_process')
 
 /**
  * Send bundle size stats to Datadog
  */
 const main = () => {
-    const retailReactAppStats = require(path.join(path.resolve(''), 'packages', 'template-retail-react-app', 'build', 'stats.json'))
-
-    const bundles = retailReactAppStats.assets
-    bundles.forEach((bundle) => {
-        const metric = `mobify_platform_sdks.bundle_size_byte`
-        const value = bundle.size
-        childProc.spawnSync('dog', ['metric', 'post', metric, value, '--host', bundle.name])
-        console.log(`${metric} ${value} --host ${bundle.name}`)
+    const buildDir = path.join(path.resolve(''), 'packages', 'pwa', 'build')
```

Comment:
```suggestion
    const buildDir = path.join(path.resolve(''), 'packages', 'template-retail-react-app', 'build')
```

Reply:
```suggestion
    const buildDir = path.resolve('packages', 'template-retail-react-app', 'build')
```
Mixing `path.join` and `path.resolve` isn't necessary.

Reply:
Good catch. I've made the changes as suggested